src/plot.js

```diff
@@ -236,6 +236,11 @@ export function plot(options = {}) {
     facetTranslate = facetTranslator(fx, fy, dimensions);
   }
 
+  // A path generator for marks that want to draw GeoJSON.
+  context.path = function () {
+    return geoPath(this.projection ?? xyProjection(scales));
+  };
+
```

Comment:
I didn’t catch this in the review, but I think this should be assigned quite a bit earlier, especially before we pass the *context* to mark initializers.

Reply:
follow-up at https://github.com/observablehq/plot/pull/2252